deps/openssl/config/archs/BSD-x86/asm/crypto/bf/bf-586.s

```diff
@@ -11,15 +11,15 @@ L_BF_encrypt_begin:
 	movl	16(%esp),%ebp
 	pushl	%esi
 	pushl	%edi
-	# Load the 2 words
+	# Load the 2 words 
```

Comment:
@mscdex I figured out why we are seeing this in the PR. It's because OpenSSL has trailing whitespace in its source, its emitted by the config process, but `git node land` strips it during landing. So, we always see the whitespace during PR diff, but it never hits the master, so then we see it again during the next PR. I'm not sure if that's a problem. I could PR some whitespace fixes up to openssl so that perhaps 1.1.1g wouldn't do this.

```
? Do you want to try reset the local master branch to upstream/master? Yes                      
⠇ Bringing upstream/master up to date...From github.com:nodejs/node                             
 * branch                    master     -> FETCH_HEAD                                           
✔  upstream/master is now up-to-date                                                            
✔  Downloaded patch to /home/sam/w/core/node/.ncu/32583/patch                                   
--------------------------------------------------------------------------------                
Applying: deps: upgrade openssl sources to 1.1.1f                                               
.git/rebase-apply/patch:542: trailing whitespace.                                               
            if(host == NULL || is_dNS_name(host))                                               
.git/rebase-apply/patch:551: trailing whitespace.                                               
 * Host dNS Name verifier: used for checking that the hostname is in dNS format                 
.git/rebase-apply/patch:935: trailing whitespace.                                               
         *                                                                                      
.git/rebase-apply/patch:944: trailing whitespace.                                               
         *                                                                                                                                                                                     
.git/rebase-apply/patch:1518: trailing whitespace.                                                                                                                                             
the given value.                                                                                
warning: squelched 10 whitespace errors                                                         
warning: 13 lines applied after fixing whitespace errors.                                       
Applying: deps: update archs files for OpenSSL-1.1.1f                                           
.git/rebase-apply/patch:326: trailing whitespace.                                               
        # Load the 2 words                                                                     
.git/rebase-apply/patch:335: trailing whitespace.                                               
        # Round 0                                                                              
.git/rebase-apply/patch:344: trailing whitespace.                                              
        # Round 1                                                                               
.git/rebase-apply/patch:353: trailing whitespace.                                              
        # Round 2                                                                              
.git/rebase-apply/patch:362: trailing whitespace.                                               
        # Round 3                                                                              
warning: squelched 2579 whitespace errors                                                       
warning: 2564 lines applied after fixing whitespace errors.                                                                                                                                    
   ✔  Patches applied                                                                                                                                                                          
There are 2 commits in the PR                                                                                                                                                                  
Please run the following commands to complete landing 
```